practice-one/src/styles/bases/variables.css

```diff
@@ -4,4 +4,19 @@
   --color-text-primary: #9e9da2;
   --color-text-secondary: #fff;
   --background-color-primary: #75bf7a;
+  --background-color-secondary: #f9f9f9;
+  --background-color-tertiary: #000;
+
+  --font-size-xsmall: 17px;
+  --font-size-small: 20px;
+  --font-size-medium: 22px;
+  --font-size-large: 24px;
+  --font-size-xlarge: 34px;
+  --font-size-xlarge2: 46px;
```

Comment:
You can use variable --font-size-xl, --font-size-2xl, --font-size-3xl .... instead of